variables.tf

```diff
@@ -43,6 +49,12 @@ variable "metrics_autoscaling" {
   default     = null
 }
 
+variable "key_pair_name" {
+  description = "The name of the key pair used by the runner to connect to the docker-machine executors."
+  type        = string
+  default     = "fleet-key"
```

Comment:
issue: All resources should be prefixed by `var.environment`, so better to calculate the default somewhere else. Or maybe the variable is not needed at all and can be hard-coded?